### DISCUSSION TITLE
Is it possible to change the moment of inertia of the UAV in each dimension to static values?

### DISCUSSION THREAD

**Discussion by dmj0013** [microsoft/AirSim#4009] [Support Q&A]
Hello! I am using airsim in my research related to the positioning of UAV's. 

I want to change the moment of inertia of the UAV in the X direction, in the Y direction and in the Z direction to static values, for example, 0.003, 0.003 and 0.015 respectively. Is there a way to edit these values in the airsim code? If so, then which variables or parameters would I need to change and in which files?

 I have been trying to change these vehicle parameters on my own, however,  I am not quite sure if they can actually be changed in airsim or not. 

Some help would mean a lot as I will be able to make progress in my research with this info.

Thanks.

**Comment by jonyMarino** [comment]
Hi @dmj0013! We have no parameters for them. You can modify the code in FastPhysicsEngine.cpp, or you can also [connect gazebo](https://microsoft.github.io/AirSim/gazebo_drone/) or jsbsim with the ExternalPhysicsEngine using AirSim as a frontend for visuals.
